controller/scheduler/formation.go

```diff
@@ -44,6 +54,15 @@ func (p Processes) IsEmpty() bool {
 	return true
 }
 
+func (p Processes) IsScaleDown() bool {
+	for _, count := range p {
+		if count < 0 {
```

Comment:
Yes makes sense, I have modified it to only prevent scaling a critical process down to zero: f79faa1